ci-operator/step-registry/origin/e2e-aws-ovn/origin-e2e-aws-ovn-workflow.yaml

```diff
@@ -9,5 +9,6 @@ workflow:
     - ref: origin-e2e-test
     post:
     - chain: ipi-deprovision
+    - chain: gather-network
```

Comment:
Why shouldn't this run on all clusters?

Reply:
@pecameron same as above, order of these two needs to be swapped so gather runs before deprovision.

Reply:
changed